test/spec/task/task.spec.ts

```diff
@@ -0,0 +1,198 @@
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import express from 'express';
+import sinon from 'sinon';
+import TaskServiceManager from '../../../src/modules/task/task-service-manager';
+import bodyParser from 'body-parser';
+import AccesstokenServiceManager from '../../../src/modules/access-token/access-token-manager';
+import ConfigService from '../../../src/modules/config/config-service';
+import TaskService from '../../../src/modules/task/task-service';
+
+chai.use(chaiHttp);
+
+let sinonSandbox: sinon.SinonSandbox;
+let appOne: any;
+let appTwo: any;
+let accessToken: string;
+const accountCreds: any = {
+  username: 'Sample Username',
+  password: 'Sample Password',
+};
+let accountId: string;
+
+describe.skip('Task Service.', () => {
+  before(async () => {
+    
+    appOne = express();
```

Comment:
At this stage, best to run monolothic app @Sparsh-Kumar  otherwise spec code becomes fat.